src/components/prediction/PredictionHeaderCard.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {Card, CardTitle} from 'react-md/lib/Cards/index';
+import SelectField from 'react-md/lib/SelectFields';
+import PropTypes from 'prop-types';
+import {modelsLabel} from '../../propTypes';
+
+class PredictionHeaderCard extends Component {
```

Comment:
not used at the moment, will have to delete if really not needed

Reply:
Why is not used predictionHeaderCard? I use it in both tabs